robocopkaka/bootcamp-29-project
# #157006899 Remove sensitive fields

#### What does this PR do?
remove sensitive fields on signup
#### Description of Task to be completed?
* remove the `isAdmin` key from the request body in the `create` method of the `users` controller
* remove the `password` field in the `response` body in the `create` method of the `users` controller
* add seed files for users, centers, categories and events
* comment out the content of the original initialisation file
* add a script to `package.json` to drop all tables, rebuild and seed with initial data
* add a `pretest` script to run migrations and seed the database before tests run
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run npm install to install all dependencies
* send a POST request to http://127.0.0.1/api/v2/users with a payload containing the following - name, email, password
* you should get a response with the user's details
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?(if applicable)
#157006899
